discord_slash/context.py

```diff
@@ -216,6 +220,9 @@ async def send(self,
             self.responded = True
         else:
             resp = await self._http.post_followup(base, self.__token, files=files)
+        if files:
+            for file in files:
+                file.close()
```

Comment:
Why would we need to close file objects? If I'm remembering correct it should be handled in http part.

Reply:
d.py implements it in `send()` https://github.com/Rapptz/discord.py/blob/757dd92272763d2b481685e50c3b3784cb3594b5/discord/abc.py#L1060